snuba/state/__init__.py

```diff
@@ -130,24 +138,40 @@ def abtest(value: Optional[Any]) -> Optional[Any]:
     return value
 
 
-def set_config(key: str, value: Optional[Any], user: Optional[str] = None) -> None:
+def set_config(
+    key: str, value: Optional[Any], user: Optional[str] = None, force: bool = False
+) -> None:
     if value is not None:
-        value = "{}".format(value).encode("utf-8")
+        enc_value = "{}".format(value).encode("utf-8")
+    else:
+        enc_value = b""
```

Comment:
Is handling of None values different to before?

Reply:
It's not. A none value is basically a delete operation. Right now, when a delete happens, we delete the key also from the config history. 

That's not great since we don't have history of the delete operation. I am going to make the change wherein the delete is also in the history. In that case I intend to show an empty byte.

Reply:
Now delete's are also saved in the log history rather than deleting the key itself.

Reply:
I wonder if there was a reason why that was happening in the first place. Will they ever be deleted from log history or do they stay there forever? Do you know what happens if we re-add the previously deleted key?

Reply:
> I wonder if there was a reason why that was happening in the first place. Will they ever be deleted from log history or do they stay there forever? Do you know what happens if we re-add the previously deleted key?
I was wrong about this. There is the audit log and and every config's last history. I have reverted that change to be like the original one